src/bindings/c/docs/api_overview.md

```diff
@@ -342,453 +425,673 @@ This strcut represents an Inference Engine entity and allows you to manipulate w
     ```
 
 
-### <a name="iecore-methods"></a>Methods
+### Methods
 
-- `IEStatusCode ie_core_get_versions(ie_core_t *core, char *device_name, ie_core_version_t *version_result)`
+- `ov_status_e ov_get_openvino_version(ov_version_t* version)`
 
-  - Description: Returns a `ie_core_version_t` with versions of the plugin specified.
+  - Description: Get version of OpenVINO.
 
   - Parameters:
 
-    - `core` -A pointer to `ie_core_t` instance.
-    - `device_name` - Name of the registered plugin.
-  - `version_result` - Dictionary mapping a plugin name .
+    - `ov_version_t` - a pointer to the version.
 
   - Return value: Status  of the operation: OK(0) for success.
 
   - Usage example:
 
     ```
-    char *xml_config_file="/localdisk/plugins/my_custom_cfg.xml";
-  char *device_name="CPU";
-    ie_core_t *ie;
-  ie_core_version_t *version;
-    IEStatusCode status= ie_core_create(xml_config_file, ie);
-    IEStatusCode status2=ie_core_get_versions(ie,device_name, version);
-    print("description:%s, major:%d, minor:%d, build_number:%s.\n",version-		  >description, version->major, version->minor, version->build_number);
+      ov_version_t version = {.description = NULL, .buildNumber = NULL};
+      ov_get_openvino_version(&version);
+      printf("description : %s \n", version.description);
+      printf("build number: %s \n", version.buildNumber);
+      ov_version_free(&version);
     ```
 
-- `IEStatusCode ie_core_load_network(ie_core_t *core, ie_network_t *network, const char *device_name,  ie_config_t config, ie_executable_network_t *exec_network_result)`
+- `ov_status_e ov_core_read_model(const ov_core_t* core, const char* model_path, const char* bin_path, ov_model_t** model)`
 
-  - Description: Loads a network that was read from the Intermediate Representation (IR) to the plugin with specified device name and creates an `ie_executable_network_t` instance of the `ie_network_t` struct.
-    You can create as many networks as you need and use them simultaneously (up to the limitation of the hardware resources).
+  - Description: Reads models from IR / ONNX / PDPD / TF / TFLite formats to create ov_model_t.
```

Comment:
```suggestion
  - Description: Reads models from IR / PyTorch / ONNX / PDPD / TF / TFLite formats to create ov_model_t.
```

Reply:
Will update these documentation issued in the next PR.